src/routes/locking/entities/holder.entity.ts

```diff
@@ -0,0 +1,15 @@
+import { Holder as DomainHolder } from '@/domain/locking/entities/holder.entity';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class Holder implements DomainHolder {
```

Comment:
What do you think about calling this `CampaignRank`? This is similar to the existing `Rank` of locking events (and we could rename that later to `LockRank` or something similar). If you prefer "holder", we could aim for `CampaignHolder` and then follow the same adjustment as above for `LockHolder`.

Reply:
Totally agree, will rename it to `CampaignRank`, thanks!

Reply:
Done in [d103261](https://github.com/safe-global/safe-client-gateway/pull/1557/commits/d103261dc2f90845eadbf8913523842a099ce968)